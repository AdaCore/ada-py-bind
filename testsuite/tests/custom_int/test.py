import gen

p = gen.Shape()
p.position.x = 490
p.position.y = 400

print p.position.x
print p.position.y

try:
    p.position.x = 2000
except Exception as e:
    print e.message

print p.position.x
