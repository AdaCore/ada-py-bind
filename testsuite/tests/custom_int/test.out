490
400
Native Ada exception: 
py_bind-types.adb:142 range check failed
Traceback: 
490
